giftcodes.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
It's good for now, and I know these are not live codes. For the next stage we can find a way to protect the list somehow.